cmd/iam-keys-check/main.go

```diff
@@ -0,0 +1,230 @@
+package main
+
+import (
+	"bytes"
+	"encoding/csv"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"log"
```

Comment:
I missed this in the first review, but we should try to the [Zap](https://godoc.org/go.uber.org/zap) structured logger for lambda functions. It looks like you already initialized it, it's just a matter of using it :) .

Reply:
done